.github/actions/configure-git-user/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Configure git user'
```

Comment:
do we need this? there are plenty of similar generic actions in the marketplace like https://github.com/marketplace/actions/configure-git-credentials

Reply:
Actually, I think we could completely replace the current propagation mechanism with an action such as this one https://github.com/marketplace/actions/create-pull-request. But I'll do it in a second iteration and only skip this action for now.